core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/jndi/DirectorySubjectNameSupport.scala

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.dsde.rawls.dataaccess.jndi
+
+import com.typesafe.config.ConfigFactory
+import org.broadinstitute.dsde.rawls.RawlsException
+import org.broadinstitute.dsde.rawls.model.{RawlsGroupName, RawlsUserSubjectId}
+
+/**
+  * Created by dvoet on 6/6/17.
+  */
+trait DirectorySubjectNameSupport {
+  private val conf = ConfigFactory.parseResources("version.conf").withFallback(ConfigFactory.load())
+
+  val directoryConfig = DirectoryConfig(
```

Comment:
It bothers me a little that this is duplicated here when it's already in Boot.scala. Is there any way that this can make use of what was already initialized?

Reply:
this is all transition code anyway... let's not get too hung up on details